### PR TITLE
F#src 3236 add biotac argument to control machine launch file

### PR DESCRIPTION
**Pull request by carebare47**
## Proposed changes

Please explain the changes you made. Add pictures or videos to explain them better if appropiate.

## Types of changes

What types of changes does your code introduce?

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

Put an x in the boxes that apply. You can also fill these out after creating the PR. This is a reminder of what we should look for before merging this code. I have:

- [x] Read and follow the [contributing guidelines](https://github.com/shadow-robot/sr_documentation/blob/master/CONTRIBUTING.md).
- [x] Checked that all tests pass with my changes
- [ ] Added tests (automatic or manual) that prove the fix is effective or that the feature works
- [ ] Added necessary documentation (if appropriate)
- [ ] Added the corresponding license to each file and add the current year of any one that you modified.
- [ ] Tested on real hardware (if appropriate)
￼

### CODE REVIEW

File: sr_robot_launch/launch/sr_left_ur10arm_hand.launch [hunk 1]
```diff
@@ -9,6 +9,8 @@
   <arg name="use_moveit" default="true"/>
```
Comment (beatrizleon) [inline sr_robot_launch/launch/sr_left_ur10arm_hand.launch]: please remove this file from this pr

File: sr_robot_launch/launch/hardware_control_loop.launch [hunk 2]
```diff
@@ -11,15 +11,17 @@
...
   <arg name="grasp_controller" default="false"/>
   <!-- Set to true to spawn the position controllers for the arm-->
   <arg name="arm_ctrl" default="true"/>
   <arg name="arm_position" default="$(eval not arm_trajectory)"/>
   <arg name="initial_z" default="0.7551"/>
   <arg name="arm_robot_hw" default="$(find sr_robot_launch)/config/$(arg side)_ur_arm_robot_hw.yaml"/>
-  <arg name="robot_description" if="$(arg hand_ctrl)" default="'$(find sr_multi_description)/urdf/$(arg side)_srhand_$(arg robot_model).urdf.xacro'"/>
-  <arg name="robot_description" unless="$(arg hand_ctrl)" default="'$(find sr_box_ur10_moveit_config)/config/ur10_with_box.urdf.xacro'"/>
+  <arg if="$(eval hand_ctrl and not biotacs)" name="robot_description" default="'$(find sr_multi_description)/urdf/$(arg side)_srhand_$(arg robot_model).urdf.xacro'"/>
+  <arg if="$(eval hand_ctrl and biotacs)" name="robot_description" default="'$(find sr_multi_description)/urdf/right_srhand_ur10_bt_sp_biotacs.urdf.xacro'"/>
```
Comment (beatrizleon) [inline sr_robot_launch/launch/hardware_control_loop.launch]: also replace the right with side

**Review by beatrizleon — CHANGES_REQUESTED** [review]
Replace the right with side

File: sr_robot_launch/launch/hardware_control_loop.launch (hunk 2 above)
Comment (carebare47) [reply sr_robot_launch/launch/hardware_control_loop.launch]: Oops, missed that, thanks :)

**Review by beatrizleon — APPROVED** [review]

**Pull request merged**